### PR TITLE
Kevin Kerner

### PR DESCRIPTION
**Pull request by kjkaustin**
This is my homework assignment on Lesson One

### CODE REVIEW

File: 02LessonTwo/homework/index.html [hunk 1]
```diff
@@ -1,10 +1,51 @@
...
+  <h2>Things I Don't Like</h2>
+      <ul>
+        <li>Liver</li>
+        <li>List element One</li>
+        <li>List element One</li>
+      </ul>
+
+  <h2>TV Shows I like to Watch</h2>
+  <figure>
+  <a href="http://www.amc.com/shows/the-walking-dead" target="_blank"><img src="https://pbs.twimg.com/profile_images/687320361201987584/4UdE2wWe_400x400.jpg"</></a>
```
Comment (victoriachuang) [inline 02LessonTwo/homework/index.html]: You might want to indent elements nested under the figure tags; once you start doing some crazy nesting, it helps to indent everything one level in!

File: 02LessonTwo/homework/index.html [hunk 2]
```diff
@@ -1,10 +1,51 @@
 <!DOCTYPE html>
```
Comment (victoriachuang) [inline 02LessonTwo/homework/index.html:1]: Great job!

File: 02LessonTwo/homework/index.html (hunk 1 above)
Comment (kjkaustin) [inline 02LessonTwo/homework/index.html]: Yep... was going to clean it up over the weekend and add the other homework elements too.  Pretty fun stuff.

Comment (victoriachuang) [inline 02LessonTwo/homework/index.html]: Oops! I got a little overeager when I was looking through the pull requests.

File: 05LessonFive/Checkpoint1/css/style.css [hunk 3]
```diff
@@ -10,34 +10,54 @@
...
+    background-size: 100%;
 }
 
 .left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+    float: right;
```
Comment (victoriachuang) [inline 05LessonFive/Checkpoint1/css/style.css:20]: what does float:right do?

File: 05LessonFive/Checkpoint1/css/style.css [hunk 4]
```diff
@@ -10,34 +10,54 @@
...
+    font-family: monospace;
 }
 
 body p {
-    font-family: monospace;
     font-size: 12px;
-    text-decoration: underline;
+    text-decoration: none;
+    color: #555;
+    padding-bottom: 20px;
```
Comment (victoriachuang) [inline 05LessonFive/Checkpoint1/css/style.css:41]: could you make the padding declarations into one statement?

File: 07LessonSeven/homework/css/style.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  width: 40%;
+  height: 250px;
+  float: left;
+  margin-top: 10px;
+  margin-left: 100px;
+  color: white;
+  font-family: sans-serif;
+}
+
+.container2 {
```
Comment (victoriachuang) [inline 07LessonSeven/homework/css/style.css:46]: can you combine with container1?

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 6]
```diff
@@ -0,0 +1,154 @@
+.clearfix:after {
+         visibility: hidden;
+         display: block;
+         font-size: 0;
+         content: " ";
+         clear: both;
+         height: 0;
+     }
+.header {
+  font-family: lato;
```
Comment (victoriachuang) [inline 08LessonEight/in-class-project/css/homework-style.css:10]: does this work? you didn't include the google fonts css or a font file

File: 11LessonEleven/in-class-project/index.html [hunk 7]
```diff
@@ -1,10 +1,73 @@
...
+  <div class="banner">
+    <h1>Order Takeaway or Delivery Food</h1>
+    <h4>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididun.
+    </h4>
+  </div>
+  <div class="icon_row">
+    <a href="#" class="button">
+      <div class="image">
+          <img src="../img/icon_home_1.svg" alt="" />
```
Comment (victoriachuang) [inline 11LessonEleven/in-class-project/index.html:18]: watch out for the file path!

**Pull request closed without merging**